deploy/scripts/longhorn-engine-exec

```diff
@@ -11,53 +14,66 @@ EXAMPLE:
 
 EOF
 
+}
 
-if [[ $# -lt 2 ]]
-then
-    echo "Require at least 2 parameters: [volume-name] and [longhorn-command-args]"
-    exit 1
-fi
+function execute_longhorn_cmd() {
+    if [[ $# -lt 2 ]]
+    then
+        echo "Error: Require at least 2 parameters: [volume-name] and [longhorn-command-args]"
+        exit 1
```

Comment:
`exit` shouldn't be used inside a function and expect the program to NOT exit.

Just call `usage()` function everytime when something is wrong with input. It's not needed if the error is e.g. `cannot find related port` etc.